iceoryx_examples/icedelivery/CMakeLists.txt

```diff
@@ -11,48 +11,48 @@ if ( NOT ICEORYX_CXX_STANDARD )
     include(IceoryxPlatform)
 endif ( NOT ICEORYX_CXX_STANDARD )
 
-add_executable(iox-ex-publisher-typed-modern ./iox_publisher_typed_modern.cpp)
-target_link_libraries(iox-ex-publisher-typed-modern
+add_executable(iox-ex-publisher-typed ./iox_publisher_typed.cpp)
+target_link_libraries(iox-ex-publisher-typed
     iceoryx_posh::iceoryx_posh
 )
-set_target_properties(iox-ex-publisher-typed-modern PROPERTIES
+set_target_properties(iox-ex-publisher-typed PROPERTIES
     CXX_STANDARD_REQUIRED ON
     CXX_STANDARD ${ICEORYX_CXX_STANDARD}
     POSITION_INDEPENDENT_CODE ON
 )
 
-add_executable(iox-ex-publisher-untyped-modern ./iox_publisher_untyped_modern.cpp)
-target_link_libraries(iox-ex-publisher-untyped-modern
+add_executable(iox-ex-publisher-untyped ./iox_publisher_untyped.cpp)
+target_link_libraries(iox-ex-publisher-untyped
     iceoryx_posh::iceoryx_posh
 )
-set_target_properties(iox-ex-publisher-untyped-modern PROPERTIES
+set_target_properties(iox-ex-publisher-untyped PROPERTIES
     CXX_STANDARD_REQUIRED ON
     CXX_STANDARD ${ICEORYX_CXX_STANDARD}
     POSITION_INDEPENDENT_CODE ON
 )
 
-add_executable(iox-ex-subscriber-typed-modern ./iox_subscriber_typed_modern.cpp)
-target_link_libraries(iox-ex-subscriber-typed-modern
+add_executable(iox-ex-subscriber-typed ./iox_subscriber_typed.cpp)
+target_link_libraries(iox-ex-subscriber-typed
     iceoryx_posh::iceoryx_posh
 )
-set_target_properties(iox-ex-subscriber-typed-modern PROPERTIES
+set_target_properties(iox-ex-subscriber-typed PROPERTIES
     CXX_STANDARD_REQUIRED ON
     CXX_STANDARD ${ICEORYX_CXX_STANDARD}
     POSITION_INDEPENDENT_CODE ON
 )
 
-add_executable(iox-ex-subscriber-untyped-modern ./iox_subscriber_untyped_modern.cpp)
-target_link_libraries(iox-ex-subscriber-untyped-modern
+add_executable(iox-ex-subscriber-untyped ./iox_subscriber_untyped.cpp)
+target_link_libraries(iox-ex-subscriber-untyped
     iceoryx_posh::iceoryx_posh
 )
-set_target_properties(iox-ex-subscriber-untyped-modern PROPERTIES
+set_target_properties(iox-ex-subscriber-untyped PROPERTIES
```

Comment:
You can write:
```
set_target_properties(iox-ex-subscriber-typed iox-ex-subscriber-untyped  
                                   iox-ex-publisher-typed iox-ex-publisher-untyped 
                                   PROPERTIES
    CXX_STANDARD_REQUIRED ON
    CXX_STANDARD ${ICEORYX_CXX_STANDARD}
    POSITION_INDEPENDENT_CODE ON
)
```
and then you can remove the three other `set_target_properties` statements above

Reply:
Done